pandas/tests/dtypes/test_dtypes.py

```diff
@@ -51,6 +51,36 @@ def test_pickle(self):
         assert result == self.dtype
 
 
+class TestBaseDtype:
+    def setup_method(self):
+        class DummyDtype(pd.core.dtypes.base.ExtensionDtype):
+            name = 'dummy'
+        self.dtype = DummyDtype()
+
+    def test_str(self):
+        assert str(self.dtype) == self.dtype.name
+
+    def test_eq(self):
+        assert self.dtype == 'dummy'
+        assert self.dtype != 'anonther_type'
+
+    def test_default_kind(self):
+        assert self.dtype.kind == 'O'
+
+    def test_construct_from_string(self):
+        dtype_instance = self.dtype.__class__.construct_from_string('dummy')
+        assert isinstance(dtype_instance, self.dtype.__class__)
+        with pytest.raises(TypeError, match="Cannot construct a 'DummyDtype' "
+                                            "from 'another_type'"):
+            self.dtype.__class__.construct_from_string('another_type')
```

Comment:
I think this would be a useful test to add to the base extension tests, and the `test_eq` as well. The defaults tests are maybe harder though.